examples/multiplatform-chat/src/clientMain/kotlin/Api.kt

```diff
@@ -51,6 +51,6 @@ suspend fun connectToApiUsingTCP(name: String): Api {
 
 private fun connector(name: String): RSocketConnector = RSocketConnector {
     connectionConfig {
-        setupPayload { Payload(name) }
+        setupPayload { buildPayload { data(name) } }
```

Comment:
Why change this line. Seems simpler before

Reply:
Old Payload constructors were removed in favor of `buildPayload`. They were really like `example constructors`. F.e. in that particular case it was `Payload(data: String, metadata: String? = null)` but for sure, such constructor will be needed in very small amount of cases. And f.e. [upper](https://github.com/rsocket/rsocket-kotlin/pull/117/files#diff-1f76e4b9227c89b5c9bc41dc524fc83469b8bc4108a4673f09a97045a045e05cR25-R28) I've used general `buildPayload` to create such narrow case constructor.
So from user perspective, similar functions can be created per use cases they need

Reply:
I think it's a matter of preference.  I'd consider using buildPayload() when I want to incrementally build the payload like buildString().  But if I have the full content already ready, even allowing string -> byte conversions, I'd like Payload(...).

Reply:
main question for me is, Do we really need functions like `Payload(String, String?)`?

anyway, may be it's better to change name of `buildPayload` ? to what? `Payload {}` ? not sure about this

Im not sure that a constructor per combination of data/metadata possible types is good.
So, if we want to support all cases with `Payload` we need 15 functions for that to combine data=String|BRP|ByteArray and metadata=String|ByteArray|BRP|Metadata|null.
In current solution those constructors can be emulated on user side, if they use it frequently with 3-4 lines of code.

Reply:
I was only asking for constructor for complete string or byte[] cases.

Reply:
Ok, will return back string/string and bytes/bytes functions